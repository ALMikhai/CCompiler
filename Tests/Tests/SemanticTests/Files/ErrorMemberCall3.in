struct b {} a1;
int *a1p;

int f() {
    int a = a1p->a;
}