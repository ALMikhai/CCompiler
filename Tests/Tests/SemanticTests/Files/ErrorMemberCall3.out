(5:13): semantic error: invalid type argument of '->'
